code/game/objects/items.dm

```diff
@@ -57,6 +57,9 @@
 	var/slowdown = 0 // How much clothing is slowing you down. Negative values speeds you up
 	var/breakouttime = 0
 
+	///list() of species types, if a species cannot put items in a certain slot, but species type is in list, it will be able to wear that item
+	var/list/species_exception = null
```

Comment:
i don't get why this is on the item, and not on the species

Reply:
its cached